### ISSUE TITLE
Statistics - are my graphs wrong for card types? 

### ISSUE THREAD

**Issue by p2635** [ankidroid/Anki-Android#4319]
###### Research

_Enter an [ x ] character to confirm the points below:_

[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum
###### Reproduction Steps
1. Open up AnkiDroid 
2. Open up statistics via left panel
3. Go to card types pie chart
###### Expected Result

Displays the pie chart according to legend (%) for the current deck.
###### Actual Result

Colours dont seem to match up to the stats (sorry for the gigantic images), I've attached images for all 3 filters which are month, year and alltime.

![alltime](https://cloud.githubusercontent.com/assets/4522927/15833635/9daa66a0-2c1f-11e6-8ebb-87505e30ab7f.png)
![month](https://cloud.githubusercontent.com/assets/4522927/15833634/9da4ba98-2c1f-11e6-9ea8-136a2a37a578.png)
![year](https://cloud.githubusercontent.com/assets/4522927/15833636/9db1f618-2c1f-11e6-8f88-5f37f853ea70.png)`
###### Debug info

Refer to the [support page](https://ankidroid.org/docs/help.html) if you are unsure where to get the "debug info".

AnkiDroid Version = 2.5.4
Android Version = 6.0.1
ACRA UUID = 125db60d-bab3-48e4-82a7-6c52f203eb46

**Comment by jvanprehn** [comment]
Does this still happen in the beta version?

**Comment by p2635** [comment]
I don't know, I'm not using the beta version (i'm just a normal user). Maybe a beta tester can answer this one?

**Comment by hssm** [comment]
Seems to be a dupe of https://github.com/ankidroid/Anki-Android/issues/3384

**Comment by jvanprehn** [comment]
It looks like this in the beta version:
![anki-card-types](https://cloud.githubusercontent.com/assets/16454627/15985293/bc3f60a8-2fe8-11e6-85b4-633fa59c2462.png)

**Comment by hssm** [comment]
The aforementioned version is now the released version. Please let us know if you still experience this problem or not so we can close it (and #3384).

**Comment by dmitry-timofeev** [comment]
It seems there is a similar or the same issue with AnkiDroid 2.6.1:
```
AnkiDroid Version = 2.6.1

Android Version = 6.0.1

ACRA UUID = be6a678a-b51b-4506-9d18-c6dd0d2189b8
```
![screenshot 21 sep 2016 23-24-48](https://cloud.githubusercontent.com/assets/7531824/18760400/89f8ff24-8109-11e6-82bd-7c5c2a064724.png)

**Comment by dequis** [comment]
Is this supposed to be fixed in 2.8.2?

![](http://dump.dequis.org/lCpso.png)
